### ISSUE TITLE
Error: failed to fill whole buffer

### ISSUE THREAD

**Issue by Napulsnik** [gcarq/rusty-blockparser#35]
Hello. Dogecoin parsing error. Ubuntu 16. Please help.
```
[06:16:43] INFO - dispatch: Status: 558752 Blocks processed. (left: 1464140, avg:   401 blocks/sec)
[06:16:53] INFO - dispatch: Status: 756529 Blocks processed. (left: 1266363, avg:   539 blocks/sec)
[06:17:03] INFO - dispatch: Status: 1121588 Blocks processed. (left: 901304, avg:   794 blocks/sec)
[06:17:13] INFO - dispatch: Status: 1623221 Blocks processed. (left: 399671, avg:  1142 blocks/sec)
[06:17:19] ERROR - worker-0: I/O Error: failed to fill whole buffer
[06:17:19] ERROR - dispatch: I/O Error: failed to fill whole buffer
```

**Comment by petermack** [comment]
I'm getting the same error, but I can't even get it to run. Tried both windows and ubuntu for windows:

> C:\Users\peter\.cargo\bin> rusty-blockparser -n -d C:\Users\peter\AppData\Roaming\Bitcoin\blocks\ unspentcsvdump C:\Users\peter\Blockdata\
> [17:56:24] INFO - main: Starting rusty-blockparser v0.6.1 ...
> [17:56:24] INFO - blkfile: Reading files from C:\Users\peter\AppData\Roaming\Bitcoin\blocks\ ...
> [17:56:24] INFO - parser: Parsing Bitcoin blockchain ...
> [17:56:24] INFO - parser: Building blockchain index ...
> [17:56:25] ERROR - worker-0: I/O Error: failed to fill whole buffer
> [17:56:25] ERROR - dispatch: I/O Error: failed to fill whole buffer
> 
> thread 'worker-1' has overflowed its stack

**Comment by jRazer** [comment]
Any news? Same error (

**Comment by Haaroon** [comment]
same issue, right now with dogecoin fails after 1506097 blocks

**Comment by gcarq** [comment]
This should be fixed with the latest release [0.7.0](https://github.com/gcarq/rusty-blockparser/releases/tag/0.7.0). Feel free to reopen if this is still an issue.
